.github/workflows/publish-rust.yml

```diff
@@ -0,0 +1,144 @@
+name: Publish Rust Crate
+
+on:
+  workflow_dispatch:
+    inputs:
+      package_path:
+        description: Path to directory with package to release
+        required: true
+        type: string
```

Comment:
Should this be a choice like the JS publish one?

Reply:
May as well